src/DS2431.h

```diff
@@ -0,0 +1,28 @@
+// 0x2D  1Kb 1-Wire EEPROM
+// tested on buspirate and two different real 1-wire masters
+
+#ifndef ONEWIRE_DS2431_H
+#define ONEWIRE_DS2431_H
+#define CRC_INIT 0xCFB5
```

Comment:
I meant that it is not in the standard, but I didn't knew about the wide-spread support for this (hence the "AFAIK").
Personnally, I prefer not using non-standard stuff, but [fair enough](http://stackoverflow.com/a/787539).